shop.py

```diff
@@ -205,6 +218,12 @@ def update_shopify(self):
             shipment.json_order = order.to_json()
             shipment.customer_phone_numbers = get_customer_phone_numbers(order)
             shipment.comment = order.note
+            for attribute in order.note_attributes:
+                for key, value in attribute.to_dict().items():
+                    if shipment.comment:
```

Comment:
This is not very pythonic. Better use:

attributes = []
for attribute in order.note_attributes:
    attributes += list(attribute.to_dict().values())
shipment.comment += '\n'.join(attribute.to_dict().values())